.github/release-drafter.yml

```diff
@@ -9,30 +9,35 @@ template: |
   $CHANGES
 exclude-labels:
   - 'kind/skip-release-notes'
-  - 'wontfix'
-  - 'triage/wont-fix'
-  - 'triage/invalid'
+  - 'dev/wont-fix'
+  - 'component/github-actions'
 categories:
-  - title: '‼️ Breaking Changes'
+  - title: '☠ Breaking Changes ☠'
```

Comment:
I fear that you might be taking the emoji game a bit far :D

Reply:
I am all for the emoji game, but I'd agree the skull might go a bit too far ;) 
How about a warning or stop sign? '⚠️ Breaking Changes' / '🚫 Breaking Changes'? 

But if you are a big fan of the skull, don't let me hold you back!

Reply:
I prefer the Warning sign with the exclamation mark. This is also what Gardener has chosen:

:warnung: Breaking Changes

https://gardener-cloud.slack.com/archives/C041L0SLZ09/p1731573289318969

Reply:
⚠️ Breaking Changes